scripts/perf/perf_config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "main_node_dir": "/Users/onyejivictor_1/Documents/code/2024/saito/saito-rust/saito-rust",
+  "spammer_node_dir": "/Users/onyejivictor_1/Documents/code/2024/saito/saito-rust/saito-spammer",
+  "verification_threads": 2,
+  "tx_size": 0,
+  "burst_count": 200
```

Comment:
these parameters should be in an array where for each item in the array, a perf test should happen